chronos_voldy_cfg/config/.version/cluster.xml

```diff
@@ -0,0 +1 @@
+0000010000000000000000
```

Comment:
@ecarlson156 what are all the `.version` files? You have a bajillion in there. If it's unintentional, use `.gitignore` 😃